2021q3/rtw88.adoc

```diff
@@ -0,0 +1,19 @@
+=== Realtek rtw88 support
+
+Links: +
+link:https://wiki.freebsd.org/WiFi/Rtw88[Rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.ORG>
+
+This project aims to bring in support for Realtek's rtw88 chips.
+
+With the growing LinuxKPI compatibility code an initial port of the Realtek rtw88 driver was done.
+This gives us a second driver symptome after Intel and helps to validate and grow the LinuxKPI code base.
```

Comment:
"symptome"? Do you mean "support for a second driver"?

Reply:
Lookup "second system syndrome"  I'll change it.

Reply:
Oh, OK.Thanks. (No need for me to look it up, I have it in cache from the last time I read TMMM.)